website/static/js/filepage/revisions.js

```diff
@@ -86,12 +90,31 @@ var FileRevisionsTable = {
             });
         };
 
+        self.showModal = function(hashName, index, hash) {
+            var showModal = bootbox.dialog({
+                title: 'Version ' + index + ' ' + hashName + ' hash',
+                message: '<div class="row">  ' +
+                    '<div class="col-md-12">' +
+                    '<div class="control-group">' +'<p class="col-xs-10">' + hash + '</p>' +
+                    '<a class="btn-sm btn-primary col-xs-2" style="align: right; text-align: center; ' +
+                    'padding: 5px; width: 80px" id="copyBtn" data-clipboard-text="' +
```

Comment:
Can we do without this hardcoded width? ("`80px`").

Also, these buttons are touching, but are not part of a button bar. They should either be separated by an appropriate margin, or they should be joined into a bar. I defer to @caneruguz as to what it should look like.

Finally, the height of these buttons is inconsistent with the height of the download button to the right